README.md

```diff
@@ -66,19 +67,19 @@ ow.actions.invoke('sample').then(result => console.log(result))
 _Client constructor supports the following mandatory parameters:_
 
 - **apihost.** Hostname and optional port for openwhisk platform, e.g. `openwhisk.ng.bluemix.net` or `my_whisk_host:80`. Used with API URL template `${protocol}://${apihost}/api/v1/`. If port is missing or port value is 443 in the apihost string, protocol is HTTPS. Otherwise, protocol is HTTP.
-- **api_key.** Authorisation key for user account registered with OpenWhisk platform.
+- **apiKey.** Authorisation key for user account registered with OpenWhisk platform.
```

Comment:
Please do not remove the constructor API parameters this will be braking API change. 
Find a way in the linter or code style rule to exclude this 

we need to keep the parameter names. It’s ok to add new aliases for the new names and mark the current ones deprecated

Reply:
ok I will do

Reply:
I added "old style" options parameters and marked them as deprecated. I did not change the readme file, because the new users use the new parameters :)

Reply:
I don't think we should be marking the current parameters as `deprecated` to fit arbitrary style rules. Changing the internal code style is fine but doing this to the constructor parameters is unnecessary and confusing. I believe that underscores are more common in JavaScript-land and much less error-prone. It's very easy to miss a capitalisation when reading documentation.